### ISSUE TITLE
Hercules InColor emulation bug

### ISSUE THREAD

**Issue by viti95** [86Box/86Box#3520]
**Describe the bug**
Hercules WriteOn! program has rendering issues:

![2023-08-06 01_08_18-86box - 86Box 4 0  build 4755](https://github.com/86Box/86Box/assets/8323882/93f3ebdc-c21d-4052-8d8b-02ebc7136153)

I've been looking for examples on how really looks, and found this capture (I don't own a real Hercules InColor)

![s-l1600](https://github.com/86Box/86Box/assets/8323882/b6a4639a-a0d2-4bfd-9a85-cca4e666b30c)

**To Reproduce**
1. Create a machine with an Hercules InColor as video card, 386 processor and 4 Mb of RAM. Install DOS 6 on it.
1. Download Hercules InColor disks from: [http://www.dosdays.co.uk/media/hercules/incolor/hercules_incolor.zip](http://www.dosdays.co.uk/media/hercules/incolor/hercules_incolor.zip)
2. Mount floppy writeon1.ima on floppy drive A:
3. Execute command "WON SAMPLE.DOC"

**Expected behavior**
The WriteOn! program should display correctly without issues

**Desktop (please complete the following information):**
 - OS: Windows 10
 - 86Box version: 4.0 build 4755 
 - Build information: old dynarec, 64-bit executable

**Additional context**
I've also tested this program on DosBox-X, the screen looks fine (no menu or border issues) but there is no color.

**Comment by Fenix770** [comment]
I tested the Windows 1.x driver for the Hercules In-Color on Windows 1.04. The machines detects the card as regular monochrome Hercules not color. Windows works but in monochrome only except for some the Windows apps that display colors.

I created this VM from using the HERCULES.DRV file included on the provided floppy disks above.
[286 Hercules In-Color Win 1.04.zip](https://github.com/86Box/86Box/files/12269178/286.Hercules.In-Color.Win.1.04.zip)


screenshot:

There are 16 colors but Windows is monochrome:
![Hercules In-Color Win 1 04 001](https://github.com/86Box/86Box/assets/31600563/bee9bf71-fdde-4d6b-9bb0-d8e2a21de178)

Windows apps can be seen in color:
![Hercules In-Color Win 1 04 002](https://github.com/86Box/86Box/assets/31600563/9e91b8a9-c88c-48a4-852f-017dcecb1880)
